pkg/client/helm.go

```diff
@@ -316,6 +317,35 @@ func (c *HelmChartClient) createPath(contextPath string) string {
 	return fmt.Sprintf("%s%s", c.consoleURL, contextPath)
 }
 
+func (c *HelmChartClient) VerifyChart(chartUrl string, values map[string]interface{}, chartSet []string) (*report.Report, error) {
+
+	helmRequest := &types.HelmVerifierRequest{
+		ChartUrl: chartUrl,
+		Chartset: chartSet,
+		Values:   values,
+	}
+	var req *http.Request
+	var err error
+	req, err = c.newRequest("POST", "/api/helm/verify", helmRequest)
```

Comment:
Prefer using existing constants instead of *magic strings*: 

```suggestion
	req, err = c.newRequest(http.MethodPost, "/api/helm/verify", helmRequest)
```